sparkey/__init__.py

```diff
@@ -122,9 +124,16 @@ def wrapper(*a, **kw):
 
 if str == bytes:
     def _to_bytes(s, name):
-        if type(s) != str:
-            raise SparkeyException(s + " must be a string")
+        if type(s) != str and type(s) != future.types.newstr:
```

Comment:
extract variable for type(s)
extract constant for future.types.newstr (`_newstr = future.types.newstr`)

Reply:
ok